cli/cli.ts

```diff
@@ -74,6 +82,13 @@ const CLI = async ({ isCI }: { isCI: boolean }) => {
 				stackName: STACK_NAME,
 			}),
 		)
+		commands.push(
```

Comment:
I have added a new command `create-health-check-device` for end-to-end health check purpose. We need to manually call this command once to enable the feature. @coderbyheart